pkg/azure/auth.go

```diff
@@ -51,5 +55,9 @@ func getAuthorizer(authLocation string, useManagedidentity bool, cpConfig *Cloud
 	}
 
 	klog.V(1).Info("Creating authorizer from Azure Managed Service Identity")
-	return auth.NewAuthorizerFromEnvironment()
+	authorizer, err := auth.NewAuthorizerFromEnvironment()
```

Comment:
A nitpick but it was probably fine to leave it as `return auth.NewAuthorizerFromEnvironment()`

Reply:
valid, functionally it would be the same thing.